examples/base/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@raincatcher/example-base",
```

Comment:
We can still add to steps to change this package.json manually to reflect actual module you want to write etc.